tests/generic_views/test_base.py

```diff
@@ -44,6 +44,18 @@ def dispatch(self, request, *args, **kwargs):
         return super().dispatch(request, *args, **kwargs)
 
 
+class SetAttributeMixin:
```

Comment:
I would put this in the one test they're used in unless you think they have potential to be used in future tests.